pkg/sbom/v1/sbom_spdx_storage_format.go

```diff
@@ -132,11 +136,15 @@ func (sbom *SBOMData) StoreFilteredSBOMName(name string) {
 	sbom.filteredSpdxData.ObjectMeta.SetName(name)
 }
 
-func (sbom *SBOMData) StoreMetadata(wlidData string) {
-	wlidLabel := map[string]string{
-		namespaceLabelKey: wlid.GetNamespaceFromWlid(wlidData),
-		kindLabelKey:      wlid.GetKindFromWlid(wlidData),
-		nameLabelKey:      wlid.GetNameFromWlid(wlidData),
+func (sbom *SBOMData) StoreMetadata(wlidData, instanceID string) {
+	metadataLabel := map[string]string{
+		namespaceLabelKey:       wlid.GetNamespaceFromWlid(wlidData),
+		kindLabelKey:            wlid.GetKindFromWlid(wlidData),
+		nameLabelKey:            wlid.GetNameFromWlid(wlidData),
+		apiVersionLabelKey:      utils.Between(instanceID, "apiVersion-", "/namespace-"),
```

Comment:
I expect an object representing the instance name or a proper parser (like we have for wild).
For now I will merge, but we should have such a parser in a global package